elasticsearch_metrics/__init__.py

```diff
@@ -1,11 +1,8 @@
 from django.apps import AppConfig
 from django.conf import settings
-
-from elasticsearch_metrics.metric import Metric
```

Comment:
Had to remove this because `Metric` can't be imported before apps have been initialized (MetricMeta uses `django.apps.apps`).